heron/common/src/cpp/network/event_loop_impl.cpp

```diff
@@ -76,7 +76,15 @@ EventLoopImpl::~EventLoopImpl() {
   event_base_free(mDispatcher);
 }
 
+static void handleTerm(evutil_socket_t _, sp_int16 what, void* ctx) {
```

Comment:
Suggest to break this change into a different PR.

Also I am wondering if it is better to move the evsignal_new() to caller in case multiple event loop objects are created in an application.